pymc3/distributions/multivariate.py

```diff
@@ -505,7 +505,13 @@ def __init__(self, n, p, *args, **kwargs):
     def _random(self, n, p, size=None):
         if size == p.shape:
             size = None
-        return np.random.multinomial(n, p, size=size)
+
+        if p.ndim > 1 and p.shape[0] > 1:
```

Comment:
i think this will still break though, though: `np.random.multinomial(1, np.array([[0.25]]))` throws `ValueError: object too deep for desired array`